ui-components/urldialog.md

```diff
@@ -104,27 +104,27 @@ window.parent.postMessage({
 }, '*');
 ```
 
-Likewise to send messages from TinyMCE back to the external page, then the `api.sendMessage()` function can be used to send messages and then in the external page an event listener can be added to receive the message:
+Similarly, to send messages from TinyMCE back to the external page, the `api.sendMessage()` function can be used to send messages, and then in the external page an event listener can be added to receive the messages:
 
 ```js
 window.addEventListener('message', function (event) {
   var data = event.data;
-  
+ 
   // Do something with the data received here
   console.log('message received from TinyMCE', data);
 });
 
 ```
 
-> Note: If possible when sending a message it's recommended to specify the target origin of where TinyMCE is running, instead of using a wildcard (`'*'`). Likewise when receiving messages also validate that `event.origin` matches the origin of where TinyMCE is running. For example if TinyMCE is running on *http://mysite.com/tinymce.html*, then if `event.origin` doesn't match `http://mysite.com` the message should be ignored.
+> Note: When sending a message it is recommended to specify the target origin of where TinyMCE is running, instead of using a wildcard (`'*'`). Similarly, when receiving messages, also validate that `event.origin` matches the origin of where TinyMCE is running. For example, if TinyMCE is running on *http://mysite.com/tinymce.html*, then if `event.origin` doesn't match `http://mysite.com` the message should be ignored.
 
 ### Supported message actions
 
-These actions are built into the URL dialog functionality and will perform an action inside the editor, based on the `mceAction` specified. The actions supported are:
+These actions are built into the URL dialog functionality and will perform an action inside the editor based on the `mceAction` specified. The actions supported are:
 
 ##### insertContent
 
-Inserts content into the editor at the current selection. The `content` property specifies what content should be inserted into the editor.
+This option inserts content into the editor at the current selection. The `content` property specifies what content should be inserted into the editor.
```

Comment:
These aren't options, so this could be confusing. Maybe `action` instead of `option`?

Reply:
Fixed